stock_picking_purchase_limit_days/models/res_config_settings.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    purchase_limit_days = fields.Char(
+        string='Purchase Limit Days',
+        help='Limit Days forward to check whether or not is necessary to buy requiered stock',)
+
+    @api.model
+    def get_values(self):
```

Comment:
I suggest to avoid these stuffs to set the parameter on the company level and a related on res.config.settings. You can find examples in Odoo code.